### ISSUE TITLE
Audit propriété d'un animal pour les ASV

### ISSUE THREAD

**Issue by PEM--** [PEM--/asv-la-soiree#21]
Ajouter les champs ci-après dans les formulaires, dans la base et dans le CSV d'extraction.
Profils concernés : ASV Diplômé + ASV Auxiliaire

>> N'apparaît que lorsque que ces types de profils sont sélectionnés.

Emplacement : En dessous de "Email"

Champs : "Êtes-vous propriétaire d'un animal ?"

Liste déroulante Oui/Non ou case à cocher (c'est toi qui voit)

Si oui : Ajouter liste de choix déroulante avec comme choix : "Chien" "Chat" "NAC"
Si non : rien

**Comment by PEM--** [comment]
Chien, chat, équidé, NAC ?

Les ASV qui ont des poneys ou des annes ou des chevaux ? Sinon, je peux mettre un "Autres".

**Comment by PEM--** [comment]
Fait. J'ai utilisé une liste de choix uniques parmi lesquels nous avons les valeurs suivantes :
* Aucun
* Chient
* Chat
* NAC
* Autres

Ces choix sont reflétés dans le CSV d'extraction des inscrits.

PS : Je déploierai ce soir.

**Comment by PEM--** [comment]
Fait et déployé.

**Comment by JulietteASV** [comment]
C'est super ! ça fonctionne bien merci

**Comment by PEM--** [comment]
Je me suis permis de mettre une petite animation lors de l'affichage du champ. Je trouve que ça fait classe et oriente bien l'utilisateur :wink:

**Comment by JulietteASV** [comment]
Ah ouai attends j'ai pas vu lol

**Comment by JulietteASV** [comment]
C'est l'apparition en fondu vers le blanc ? C'est bien et en effet ça fait
class ^^

**Comment by PEM--** [comment]
Yes et le fait d'augmenter la hauteur du formulaire pour bien montrer à l'utilisateur qu'une donnée supplémentaire est requise du fait de son choix précédent.

**Comment by JulietteASV** [comment]
oui c great
